bundles/org.openhab.binding.remoteopenhab/src/main/java/org/openhab/binding/remoteopenhab/internal/RemoteopenhabHandlerFactory.java

```diff
@@ -70,16 +170,36 @@ public boolean supportsThingType(ThingTypeUID thingTypeUID) {
         return SUPPORTED_THING_TYPES_UIDS.contains(thingTypeUID);
     }
 
+    @Override
+    public @Nullable Thing createThing(ThingTypeUID thingTypeUID, Configuration configuration,
+            @Nullable ThingUID thingUID, @Nullable ThingUID bridgeUID) {
```

Comment:
The purpose is to inject the bridgeId in the thing UID. You can find the same method in a lot of bindings. Is it something no more needed because already done by the core framework ?

Reply:
It is still needed I think. I was just curious.